backend/worker/pe_scripts/sixgill/README.md

```diff
@@ -0,0 +1,12 @@
+# Quick ReadMe for PE!!!
+
+1. app/config/config.ini = put your sixgill credentials here....
```

Comment:
let's make a README.md in pe_scripts just explaining what P&E is, maybe link to the P&E repo

Reply:
Addressed here https://github.com/cisagov/crossfeed/pull/1413/commits/2223c7dcc0b9db1f92e5605a3608b4ee38d0ae5f